shared/js/smart-screen/card_manager.js

```diff
@@ -661,13 +661,13 @@
         name = {
           raw: card.name.raw
         };
+      } else if (card.name && card.name.id) {
+        name = card.name;
```

Comment:
You didn't modify the comment above with respect to the change.
Please modify it and change reviewer to Tzu-lin since the code here is written by him.